streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -239,7 +246,7 @@ public void run() {
         } catch (Exception e) {
             // we have caught all Kafka related exceptions, and other runtime exceptions
             // should be due to user application errors
-            log.error(String.format("stream-thread [%s] Streams application error during processing: ", this.getName()),  e);
+            log.error(logPrefix + " Streams application error during processing: ",  e);
```

Comment:
Thanks for the pointer, I thought it does not support it since when I did that on Intellij it references to this function:

```
public void error(String format, Object arg1, Object arg2);
```

But maybe it's just an IntelliJ issue.